### PR TITLE
Add black linting package and remove autopep8

### PR DESCRIPTION
**Pull request by gsheni**
- Add black linting package and remove autopep8
- Add Makefile commands

### CODE REVIEW

File: dev-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+codecov==2.1.12
+flake8==3.9.2
+autopep8==1.5.7
```
Comment (rwedge) [inline dev-requirements.txt:3]: do we still need autopep8?

Comment (gsheni) [reply dev-requirements.txt]: Thanks, also realized I didn't remove it in Woodwork. Removed here and there.

File: .github/workflows/unit_tests_with_latest_deps.yml [hunk 2]
```diff
@@ -40,10 +40,10 @@ jobs:
         run: pip install -r test-requirements.txt
       - if: ${{ matrix.python_version != 3.8 || matrix.featuretools_version != 'Main' }}
         name: Run unit tests (no code coverage)
-        run: make unit-tests
+        run: make tests
```
Comment (rwedge) [inline .github/workflows/unit_tests_with_latest_deps.yml:43]: in woodwork makefile use `test`

File: dev-requirements.txt [hunk 3]
```diff
@@ -0,0 +1,4 @@
+codecov==2.1.12
+flake8==3.9.2
```
Comment (rwedge) [inline dev-requirements.txt:2]: should we bump flake8 to 4.0.1 at the same time?

Comment (gsheni) [reply dev-requirements.txt]: Fixed

File: .github/workflows/unit_tests_with_latest_deps.yml (hunk 2 above)
Comment (gsheni) [reply .github/workflows/unit_tests_with_latest_deps.yml]: fixed

**Review by rwedge — APPROVED** [review]
LGTM

**Pull request merged**